pkg/manager/member/pump_member_manager.go

```diff
@@ -256,14 +255,6 @@ func getNewPumpConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 		"pump-config": confTextStr,
 	}
 
-	if basePumpSpec.ConfigUpdateStrategy() == v1alpha1.ConfigUpdateStrategyRollingUpdate {
-		sum, err := Sha256Sum(data)
-		if err != nil {
-			return nil, err
-		}
-		suffix := fmt.Sprintf("%x", sum)[0:7]
-		name = fmt.Sprintf("%s-%s", name, suffix)
-	}
```

Comment:
why remove this block? A suffix should be added to the configmap name when `ConfigUpdateStrategyRollingUpdate` configured.